pkg/receiver/telegrafreceiver/receiver.go

```diff
@@ -105,6 +110,37 @@ func (r *telegrafreceiver) Start(ctx context.Context, host component.Host) error
 	return err
 }
 
+// Consume metrics and retry on recoverable errors
+func (r *telegrafreceiver) consumeWithRetry(ctx context.Context, metrics pdata.Metrics) error {
+	constantBackoff := backoff.WithMaxRetries(backoff.NewConstantBackOff(r.consumeRetryDelay), uint64(r.consumeMaxRetries))
+
+	// retry handling according to https://github.com/open-telemetry/opentelemetry-collector/blob/master/component/receiver.go#L45
+	fErr := backoff.RetryNotify(
```

Comment:
what does this mean `fErr`? Can we make the name more desriptive?

Reply:
What do you suggest :) ? `retryNotifyErr`? The scope of this func is pretty small (nice), no need IMHO to make it any more explicit/clear